src/compiler/python_generator.cc

```diff
@@ -657,7 +658,7 @@ bool PrivateGenerator::PrintServiceClass(
           out->Print("options, channel_credentials,\n");
           out->Print(
               "insecure, call_credentials, compression, wait_for_ready, "
-              "timeout, metadata)\n");
+              "timeout, metadata, _registered_method=True)\n");
```

Comment:
Looks like this pushes the generated line beyond 80 characters. Can you reformat the generated line so we stay below 80?